### PR TITLE
linux/u*: add  links

### PR DESCRIPTION
**Pull request by CleanMachine1**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/unix2dos.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 > Change Unix-style line endings to DOS-style.
 > Replaces CR with CRLF.
+> More information: <https://manned.org/unix2dos>.
```
Comment (marchersimon) [inline pages/linux/unix2dos.md:5]:
```suggestion
> More information: <https://waterlan.home.xs4all.nl/dos2unix.html>.
```

File: pages/linux/unix2mac.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 > Change Unix-style line endings to macOS-style.
 > Replaces CR with LF.
+> More information: <https://manned.org/unix2mac>.
```
Comment (marchersimon) [inline pages/linux/unix2mac.md:5]:
```suggestion
> More information: <https://waterlan.home.xs4all.nl/dos2unix.html>.
```

**Review by marchersimon — APPROVED** [review]
Thanks @CleanMachine1

**Review by SethFalco — APPROVED** [review]

**Comment by CleanMachine1** [comment]
@navarroaxel could you review so I can merge this?

**Pull request merged**